src/Core/Domain/Uuid.ts

```diff
@@ -8,4 +8,4 @@ export const getUuidFromRaw = (raw: string): Uuid => ({
   short: raw.substring(0, 4),
 });
 
-export const getShortUuidFromRaw = (raw: string): string => raw.substring(0, 4);
+export const getShortUuidFromRaw = (raw: string): string => raw.substring(0, 8);
```

Comment:
add docstrings ;)

Reply:
I will, but I thought this is self-explanatory :D